guides/common/modules/proc_upgrading-a-disconnected-project-server.adoc

```diff
@@ -180,12 +155,11 @@ include::snip_steps-needs-reboot.adoc[]
 # {foreman-maintain} service restart
 ----
 
-. If you have the OpenSCAP plugin installed, but do not have the default OpenSCAP content available, enter the following command.
+. If you have the OpenSCAP plugin installed but do not have the default OpenSCAP content available, enter the following command.
 +
 [options="nowrap" subs="attributes"]
 ----
 # foreman-rake foreman_openscap:bulk_upload:default
 ----
 
-. In the {ProjectWebUI}, navigate to *Configure* > *Discovery Rules*.
- Associate selected organizations and locations with discovery rules.
+. In the {ProjectWebUI}, navigate to *Configure* > *Discovery Rules* and associate the selected organizations and locations with the appropriate discovery rules.
```

Comment:
In https://github.com/theforeman/foreman-documentation/pull/3907/files#r2120504984 and https://github.com/theforeman/foreman-documentation/pull/3907/files#r2120550390, there were suggestions that we can drop one or both of these steps entirely.